Yoast/javascript
# Add rules for stemming exceptions on ter- and keter-

## Summary

This PR can be summarized in the following changelog entry:
* [yoastseo] Adds 
a) check for words-exceptions on _ter-_ where _ter-_ must not be stemmed 
b) check for words-exceptions on _keter-_ in which _ter-_ must not be stemmed while suffix _ke-_ must be stemmed.
* [Yoast SEO Premium] Improves keyphrase recognition for words (of different word categories) on _ter-_ and _keter-_.

## Test instructions

This PR can be tested by following these steps:

* Run yarn test and make sure all current specs pass
* In Indonesian Premium: 
a) Add words from the [doNotStemTer](https://github.com/Yoast/YoastSEO.js-premium-configuration/pull/187/files) exception list in the morphology data file to the keyphrase and to the title/text/meta description and see if those are currently stemmed. For example, add _teritorial, terampil, terenyuh_. If _ter-_ is not removed during stemming, the check is successful. 
b) Add ke- to any words from doNotStemTer list to the keyphrase and to the title/text/meta description and see if those are correctly stemmed (_ke-_ must be removed, but _ter-_ must not).

## Quality assurance


Fixes https://yoast.atlassian.net/jira/software/projects/LIN/boards/4?selectedIssue=LIN-392
